backend/src/asiakirja/abstractPdf.ts

```diff
@@ -8,9 +8,17 @@ import PDFKitReference = PDFKit.PDFKitReference;
 const INDENTATION_BODY = 186;
 
 export abstract class AbstractPdf {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   private title: string;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   private fileName: string;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   protected fileBasePath: string;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   protected doc: PDFKit.PDFDocument;
```

Comment:
Näissä oli se ongelma, ettei ollut konstruktoria, jossa nää initialisoitaisiin. Sellaisen voisi tehdä?

Reply:
Ei ole järkeä alustaa kaikkia proppareitaaaa konstruktorissa. Lissäsin huutarit noihin, näyttäis kelpaavan.